backend/embedding/loaders/s3.py

```diff
@@ -1,9 +1,16 @@
 import os
 import tempfile
-
+import logging
```

Comment:
in this file. it was used `print()` as a log message.
I modified to use logging